apps/src/templates/manageStudents/ManageStudentsNameCell.jsx

```diff
@@ -43,7 +43,9 @@ class ManageStudentNameCell extends Component {
         {this.props.isEditing &&
           <div style={{paddingTop: 10}}>
             <input value={editedValue} onChange={this.onChangeName}/>
-            <div style={{color: 'red', fontSize: 9, height: 10}}>{this.props.editedValue.length === 0 ? 'required' : ''}</div>
+            <div style={{color: 'red', fontSize: 9, height: 10}}>
```

Comment:
any reason to do this in-line rather than as a style? just because it's the only one?

Reply:
yep - although I should pull it out - it'd be cleaner. I'll add to another PR.